.github/workflows/build_and_test.yml

```diff
@@ -201,6 +173,92 @@ jobs:
         name: unit-tests-log-${{ matrix.modules }}-${{ matrix.comment }}-${{ matrix.java }}-${{ matrix.hadoop }}-${{ matrix.hive }}
         path: "**/target/unit-tests.log"
 
+  pyspark:
+    name: "Build modules: ${{ matrix.modules }}"
+    runs-on: ubuntu-20.04
+    container:
+      image: dongjoon/apache-spark-github-action-image:20201015
```

Comment:
As we have `container`, do we still need `runs-on`?

Reply:
Yes. It's required. Many people complains about that. :)
- https://github.community/t/confused-with-runs-on-and-container-options/16258

Reply:
interesting... :)